### PR TITLE
[Workers] jamstack app tutorial revision

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: products/workers/src/content/tutorials/build-a-jamstack-app/index.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ $ wrangler generate todos
...
+Wrangler templates are Git repositories. You can create your own templates or use one from the [Template Gallery](/examples).
 
 Wrangler’s default template includes support for building and deploying JavaScript-based projects, including Webpack support. Inside of your new `todos` directory, `index.js` represents the entry point to your Cloudflare Workers application.
 
-All Cloudflare Workers applications start by listening for `fetch` events, which are triggered when a client makes a request to a Workers route. When that request occurs, your constructed response will be returned to the user. This tutorial will guide you through understanding how the request/response pattern works, and how you can use it to build fully-featured applications.
+All Cloudflare Workers applications start by listening for `fetch` events, which are triggered when a client makes a request to a Workers route. When that request occurs, your constructed response will be returned to the user. This tutorial will guide you through understanding how the request/response pattern works and how you can use it to build fully-featured applications.
```
Comment (deadlypants1973) [inline products/workers/src/content/tutorials/build-a-jamstack-app/index.md:45]:
```suggestion
All Cloudflare Workers applications start by listening for `fetch` events, which are triggered when a client makes a request to a Workers route. When that request occurs, your constructed response will be returned to the user. This tutorial will guide you through understanding how the request/response pattern works and how you can use it to build fully featured applications.
```

File: products/workers/src/content/tutorials/build-a-jamstack-app/index.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ pcx-content-type: tutorial
 
 import TutorialsBeforeYouStart from "../../_partials/_tutorials-before-you-start.md"
 
-# Build a Todo list JAMstack application
+# Build a todo list JAMstack application
```
Comment (lukeed) [inline products/workers/src/content/tutorials/build-a-jamstack-app/index.md:10]: this should remain capitalized

File: products/workers/src/content/tutorials/build-a-jamstack-app/index.md [hunk 3]
```diff
@@ -123,7 +125,7 @@ async function handleRequest(request) {
 }
 ```
 
-Workers KV is an eventually consistent, global datastore. In other words, any writes within a region are immediately reflected within that same region, but will not be immediately available in other regions. However, those writes will eventually be available everywhere and, at that point, Workers KV guarantees that data within each region will be consistent.
+Workers KV is an eventually consistent, global datastore. Any writes within a region are immediately reflected within that same region but will not be immediately available in other regions. However, those writes will eventually be available everywhere and, at that point, Workers KV guarantees that data within each region will be consistent.
```
Comment (marciocloudflare) [inline products/workers/src/content/tutorials/build-a-jamstack-app/index.md:128]: What does it mean to be "eventually consistent"? Is it in the sense that it becomes consistent in the end, after many tries? Is it that it's consistent from an event point of view? I'm not sure this is totally clear.

Comment (marciocloudflare) [inline products/workers/src/content/tutorials/build-a-jamstack-app/index.md:128]: Should we capitalize "Eventual Consistent" so it's clear that it's a type of database, not one that ends up being consistent at the end? Right now it seems we're saying that this database is, after many tries or by the end, consistent.

**Review by marciocloudflare — APPROVED** [review]

File: products/workers/src/content/tutorials/build-a-jamstack-app/index.md (hunk 2 above)
Comment (deadlypants1973) [inline products/workers/src/content/tutorials/build-a-jamstack-app/index.md:10]:
```suggestion
# Build a Todo list JAMstack application
```

**Pull request merged**